src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/grid_actions.html.twig

```diff
@@ -22,13 +22,12 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  *#}
-
 {% if grid.actions.grid|length > 0 %}
-  {% set girdActionsButtonId = grid.id-grid ~ '-actions-button' %}
+  {% set gridActionsButtonId = grid.id ~ '-grid-actions-button' %}
```

Comment:
funny thing: I'll be able to detect this issue on Survival tests *only*. In dev mode, we should be allowed to ask Twig to be really strict...